.github/workflows/transports-ci.yml

```
@@ -0,0 +1,200 @@
+
+      # Authenticate with Docker Hub for pushing images
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      # Build and push Docker image using the freshly built UI static files
+      - name: Extract metadata for Docker
+        id: meta
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:5:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)